libs/components/lookup/src/lib/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -199,8 +199,8 @@ export class SkyAutocompleteInputDirective
   }
 
   public ngOnDestroy(): void {
-    this._blur.complete();
-    this._textChanges.complete();
+    this.#_blur.complete();
+    this.#_textChanges.complete();
```

Comment:
This seems wrong; I'd only expect properties that start with an underscore to be referenced in property accessors, hence the pattern. I'd expect any other private field not exclusively used inside property accessors to not have an underscore prefix. There are lots of these instances across the files in the PR.